flake/check.nix

```diff
@@ -1,6 +1,53 @@
 { self, ... }: {
   perSystem = { config, self', inputs', pkgs, system, ... }: {
     packages = {
+      all-ci-packages = pkgs.linkFarmFromDrvs "all-ci-packages"
```

Comment:
🧸️https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#grouping-log-lines 

how happy CI users will find errors why something wrong? logs could be huge.

Reply:
Excellent idea to group them, not sure how I would cleanly inject it into the build process though. Any suggestions?

Reply:
wrap each package into https://nixos.org/manual/nix/stable/language/builtins.html#builtins-trace

Reply:
Sounds good, but how well will it work within the actual logs, since it prints output from each builts in parallel

Reply:
that is why i am🧸️.  is it million lines of logs intermixed? how people will use CI to see anything?

Reply:
I am personally OK. I can read any logs. But I have seen that most people do not.

Reply:
Yeah I understand, I guess we'll just need to teach people to read logs. So far from errors i've encountered it was pretty obvious from which check it was coming, so I think it is possible to teach them.

Reply:
Else we can look at if it's possible to dump them into a log file, and then group them by built derivation afterwards